server/controllers/finance/reports/generalLedger/index.js

```diff
@@ -0,0 +1,84 @@
+
+/**
+ * @overview General Ledger Accounts Reports
+ *
+ * @description
+ * This module contains all the code for rendering PDFs of Journal
+ */
+
+const _ = require('lodash');
+const ReportManager = require('../../../../lib/ReportManager');
+const GeneralLedger = require('../../generalLedger');
+const AccountReport = require('../reportAccounts');
+const Accounts = require('../../accounts');
+
+const REPORT_TEMPLATE = './server/controllers/finance/reports/generalLedger/report.handlebars';
+const ACCOUNT_SLIP_TEMPLATE = './server/controllers/finance/reports/generalLedger/accountSlip.handlebars';
+
+exports.report = renderReport;
+exports.accountSlip = renderAccountSlip;
+
+
+/**
+ * GET reports/finance/general_ledger
+ *
+ * @method report
+ */
+function renderReport(req, res, next) {
+  const options = _.extend(req.query, { filename: 'TREE.GENERAL_LEDGER', csvKey: 'rows' });
+  let report;
+  let data;
+
+  try {
+    report = new ReportManager(REPORT_TEMPLATE, req.session, options);
+  } catch (e) {
+    return next(e);
+  }
+
+  return GeneralLedger.getlistAccounts()
+    .then((rows) => {
+      data = { rows };
```

Comment:
I don't think you need this extra variable `data`.  It's not a problem to have it defined, but I kept looking for where it was needed.  Removing it would clarify the code:
```js
.then((rows) => return report.render({ rows }))
```